apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -31,17 +31,10 @@ function globalEventPropsHandler( block ) {
 		return {};
 	}
 
-	// Pick up variation slug from `core/embed` block.
-	if ( block.name === 'core/embed' && block?.attributes?.providerNameSlug ) {
-		return { variation_slug: block.attributes.providerNameSlug };
-	}
-
-	// Pick up variation slug from `core/social-link` block.
-	if ( block.name === 'core/social-link' && block?.attributes?.service ) {
-		return { variation_slug: block.attributes.service };
-	}
-
-	return {};
+	return {
+		variation_slug: select( 'core/blocks' ).getActiveBlockVariation( block.name, block.attributes )
```

Comment:
Is that something that depends on a specific Gutenberg version being available, perhaps?

Reply:
Yes, that's a relatively new feature in Gutenberg. Opening a PR to fix this.